plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -166,20 +192,25 @@ def main():
     l3out = module.params.get("l3out")
     l3out_schema = module.params.get("l3out_schema")
     l3out_template = module.params.get("l3out_template")
+    l3out_template_id = module.params.get("l3out_template_id")
     l3out_on_apic = module.params.get("l3out_on_apic")
     route_reachability = module.params.get("route_reachability")
+    l3out_uuid = module.params.get("l3out_uuid")
+    l3out_on_l3out_template = module.params.get("l3out_on_l3out_template")
```

Comment:
do we need this option, or can we do lookup and when l3out template not found in schema it does a lookup in templates?